examples/ansibleee-play.yaml

```diff
@@ -4,33 +4,26 @@ metadata:
   name: ansibleee-play
   namespace: openstack
 spec:
-  # We can specify either playbook, which will run with default ansible-runner options,
-  # or args, which allows specify the whole command that we want to execute
   play: |
     - name: Print hello world
       hosts: all
       tasks:
       - name: Using debug statement
         ansible.builtin.debug:
           msg: "Hello, world this is ansibleee-play.yaml"
-  # playbook: "test.yaml"
-  # args: ["ansible-runner", "run", "/runner", "-vvv", "-p", "test.yaml"]
   image: "quay.io/jlarriba/openstack-tripleo-ansible-ee:stable"
-  inventory: |
```

Comment:
Since we define the play in-line in this example do we want to leave the in-line inventory too?

Reply:
Yes, why not. This was intended as the most customizable example: play, inventory and image